### ISSUE TITLE
Method touch() doesn't resave the session

### ISSUE THREAD

**Issue by jjjuk** [mgcrea/fastify-session#5]
### Renew expiries please!

Hey! Hello code! Seems like `touch()` is not working. No matter where I place it in code. Session puts new data in store with `save()` but does not renew expiries of key and cookie.

I guess `touch()` is the method I looking for, it suppose to be same as express-session's one

```typescript
app.decorateRequest('authorize', async function (user: UserAttributes) {
    const req: FastifyRequest = this
    await req.session.touch() // not working
    req.session.data.user = user
    req.session.data.auth = true
    await req.session.save()
    await req.session.touch() // pathetic attempt
  })
```

I do it with decorator, cool thing. 

Please Dear @mgcrea help me with renewing sessions! I don't want to switch to other package((

**Comment by mgcrea** [comment]
Hey! Thanks for pointing that out, indeed the touch functionality was missing and I think I properly did add it along with unit tests to prevent any regression.

Current code should also now properly auto-renew sessions maxAge on every request (touched when restored from cookie). Tell me if the current behaviour works for you.

Just released `v0.14.0`.

**Comment by jjjuk** [comment]
@mgcrea Thank you very much, everything is working fine now!
